repos/system_upgrade/el7toel8/libraries/dnfplugin.py

```diff
@@ -155,7 +155,7 @@ def install_initramdisk_requirements(packages, target_userspace_info, used_repos
             '--setopt=module_platform_id=platform:el8',
             '--setopt=keepcache=1',
             '--disablerepo', '*'
-        ] + repos_opt + [package for package in packages]
+        ] + repos_opt + packages[:]
```

Comment:
here definitelly do not copy the the list.

Reply:
So, how can we solve the `Unnecessary use of a comprehension` here,  any idea?

Reply:
`['dnf', 'install', ...] + repos_opt + packages`?

Reply:
@fernflower  I'm affraid to do that, because, if we change `packages` in other part of code, will reflect on these piece of code... copying, I just make sure we do not have any changes by another code... I really don't know how the right approach  here... :)

Reply:
@fellipeh but we are not changing packages, are we?

Reply:
@fellipeh that's not true, as during the merge operation you are creating completely new object. So any change in packages doesn't affect the new object at all.
```python
>>> aaa = range(3)
>>> bbb = ['a', 'b', 'c']
>>> ccc = aaa  # ccc is now the same object as aaa
>>> ddd = aaa + bbb # ddd is completely new object
>>> aaa.pop()
2
>>> print(aaa)
[0, 1]
>>> print(ccc)
[0, 1]
>>> print(ddd)
[0, 1, 2, 'a', 'b', 'c']
```